src/RemoteRequest/CachedRemoteGetRequest.php

```diff
@@ -28,7 +30,7 @@ final class CachedRemoteGetRequest implements RemoteGetRequest {
 	 *
 	 * @var string
 	 */
-	const TRANSIENT_PREFIX = 'amp_remote_request_';
+	const TRANSIENT_PREFIX = 'amp_remote_request_v2_';
```

Comment:
Updated the transient prefix to bust the cache.

Reply:
Might not be necessary if we check the type of the class per my comment below.